easybuild/easyconfigs/t/Trinity/Trinity-2.6.5-foss-2017a.eb

```diff
@@ -0,0 +1,34 @@
+
+name = 'Trinity'
+version = '2.6.5'
+
+homepage = 'https://github.com/trinityrnaseq/trinityrnaseq/wiki'
+description = """Trinity assembles transcript sequences from Illumina RNA-Seq data."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+tnamever = '%(namelower)srnaseq-%(version)s'
+source_urls = ['https://github.com/trinityrnaseq/trinityrnaseq/archive/']
+sources = ['%(name)s-v%(version)s.tar.gz']
+
+checksums = ['fe8e056ce40e5e3056d7f71cf95a2ccc6e11f6ec60867f56769ba463baae5b43']
+
+builddependencies = [
+    ('Autotools', '20150215'),
+]
+
+dependencies = [
+    ('Java', '1.8.0_162', '', True),
+    ('ant', '1.10.1', '-Java-%(javaver)s', True),
+    ('Bowtie', '1.2.2'),
+    ('Bowtie2', '2.3.2'),
+    ('ncurses', '6.0'),
+    ('zlib', '1.2.11'),
+    ('Perl', '5.24.1'),
+    ('SAMtools', '1.6'),
+    ('Jellyfish', '2.2.6'),
+    ('Salmon', '0.9.1'),
+    ('BLAST+', '2.6.0', '-Python-2.7.13'),
```

Comment:
@ebirn @timeu We don't have these 4 dependencies in our most recent `Trinity` easyconfig (`Trinity-2.6.6-intel-2018a.eb`).
Can you clarify why these have been added? Should we add them to `Trinity-2.6.6-intel-2018a.eb` as well (if so, we'll do that in a separate PR)?

Reply:
Yes I believe these tools should be added. They are at least optional dependencies:

1) SAMtools, see https://github.com/trinityrnaseq/trinityrnaseq/search?q=samtools&unscoped_q=samtools
more specifically, for example: direct call to samtools: https://github.com/trinityrnaseq/trinityrnaseq/blob/e678e74a99c3c1310968eb273d90ed259886f917/util/retrieve_sequences_from_fasta.pl#L14
2) Jellyfish: https://github.com/trinityrnaseq/trinityrnaseq/search?p=2&q=jellyfish&unscoped_q=jellyfish
called from Trinity directly, used in Dockerfile setup, as well as ugly Bioconda path replacement: https://github.com/trinityrnaseq/trinityrnaseq/blob/ec9896fcf422bc21b8558f13b0f8c2f80fa236f4/bioconda_recipe/trinity/2.6.6/build.sh
3) Salmon: https://github.com/trinityrnaseq/trinityrnaseq/search?q=salmon&unscoped_q=salmon
referenced in Changelog, code, Dockerfile, Travis env setup
4) BLAST+: various references: https://github.com/trinityrnaseq/trinityrnaseq/search?q=blast%2B&unscoped_q=blast%2B
also used in Dockerfile setup, even though I can't seem to find a reference in the other sources, where i.e. blastn is called directly

Maybe on 4) somebody else can chime in? I'm not 100% on this

Reply:
@ebirn Thanks for clarifying, it doesn't hurt to add these deps, indeed (even BLAST+, if you see hints about it being used).

Are you up for opening a PR to add these to `Trinity-2.6.6-intel-2018a.eb`?